tools/bridge/tests/test_config.py

```diff
@@ -3,18 +3,64 @@
 from bridge.config import validate_config
 
 
-def test_validate_missing_keys():
+@pytest.fixture(scope="session")
+def valid_config():
+    return {
+        "home_rpc_url": "url",
+        "foreign_rpc_url": "url",
+        "token_contract_address": "0x4B0b6E093a330c00fE614B804Ad59e9b0A4FE8A9",
+        "foreign_bridge_contract_address": "0xbb1046b0fe450aa48deaff6fa474adbf972840dd",
+    }
+
+
+def test_validate_missing_keys(valid_config):
+    invalid_config = {**valid_config}
+    del invalid_config["home_rpc_url"]
```

Comment:
suggestion: use https://toolz.readthedocs.io/en/latest/api.html#toolz.dicttoolz.dissoc

```
invalid_config = dissoc(valid_config, "home_rpc_url")
```

Reply:
Thanks for this nice hint. Like it much more. I should remember this library more often. :+1: